src/resources/bootstrap.cpp

```diff
@@ -300,6 +235,63 @@ static bool copy_file(const Path &source_path, const Path &target_path, Bootstra
     }
 }
 
+static bool remove_directory_recursive(Path &path) {
```

Comment:
I'd recommend using `[[nodiscard]]` for such an important operation ... and probably at other spots as well

Reply:
Good idea! Added :)